f/pabstMarkers/fn_getGroupMarkerStyle.sqf

```diff
@@ -1,175 +1,176 @@
+private ["_red","_green","_blue","_yellow","_orange","_black","_pink","_white","_fireteam","_unknown","_hq","_support","_supportAT","_recon","_mortar","_maint","_mech","_armor","_air","_plane","_artillery","_med","_size","_input","_texture","_color"];
```

Comment:
This is kind of a thing that's on plan, recommended change would be to revert this to upstream/master version of it and apply your changes to that.
Minimal changes are almost always the best way to go.
This also goes for the other file.